dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ProcessDefinitionServiceImpl.java

```diff
@@ -1278,7 +1278,8 @@ private String recursionProcessDefinitionName(long projectCode, String processDe
             putMsg(result, Status.SUCCESS);
         } catch (Exception e) {
             result.put(Constants.STATUS, Status.REQUEST_PARAMS_NOT_VALID_ERROR);
-            result.put(Constants.MSG, e.getMessage());
+            putMsg(result, Status.REQUEST_PARAMS_NOT_VALID_ERROR, "");
+            logger.error(Status.REQUEST_PARAMS_NOT_VALID_ERROR.getMsg(), e);
```

Comment:
I think use `Status.INTERNAL_SERVER_ERROR_ARGS` might be better.
```java
putMsg(result, Status.INTERNAL_SERVER_ERROR_ARGS, e.getMessage());
```
And it's suggested to find out why there is an exception here.

Reply:
I'll fix it.